jetbrains-core/src/software/aws/toolkits/jetbrains/core/AwsClientManager.kt

```diff
@@ -57,4 +57,8 @@ open class AwsClientManager(project: Project, sdkClient: AwsSdkClient) :
     }
 }
 
-inline fun <reified T : SdkClient> Project.awsClient(): T = AwsClientManager.getInstance(this).getClient()
\ No newline at end of file
+inline fun <reified T : SdkClient> Project.awsClient(): T = AwsClientManager.getInstance(this).getClient()
+
+inline fun <reified T : SdkClient> Project.awsClient(region: AwsRegion): T = AwsClientManager
```

Comment:
We could just add region as a default parameter to the above extension function. Eg:

`awsClient(regionOverride: Region? = null)`